src/d_main.c

```diff
@@ -495,19 +496,8 @@ void D_PageDrawer(void)
 {
   if (pagename)
     {
-      int l = W_CheckNumForName(pagename);
-      byte *t = W_CacheLumpNum(l, PU_CACHE);
-      size_t s = W_LumpLength(l);
-      unsigned c = 0;
-      while (s--)
-	c = c*3 + t[s];
-      V_DrawPatchFullScreen((patch_t *) t);
-      if (c==2119826587u || c==2391756584u)
-        // [FG] removed the embedded DOGOVRLY title pic overlay graphic lump
-        if (W_CheckNumForName("DOGOVRLY") > 0)
```

Comment:
I'm not sure if this easter-egg has ever been used, but I'd rather have this checked and be sure than throwing it out altogether.